src/Grid/grid.jl

```diff
@@ -708,25 +711,11 @@ end
 
 function Base.show(io::IO, ::MIME"text/plain", grid::Grid)
     print(io, "$(typeof(grid)) with $(getncells(grid)) ")
-    typestrs = sort!(collect(Set(celltypes[typeof(x)] for x in grid.cells)))
-    str = join(io, typestrs, '/')
+    typestrs = sort!(collect(Set(repr(typeof(x)) for x in grid.cells)))
```

Comment:
This seems to be really slow, try showing something like `generate_grid(Triangle, (1000, 1000))` in the REPL.

Reply:
Sorry, missed that performance regression. Fixed in #559.